components/brave_wallet_ui/page/screens/swap/components/swap/routes/routes.tsx

```diff
@@ -124,12 +124,16 @@ export const Routes = (props: Props) => {
         gap='8px'
         padding='2px 16px'
       >
-        {sortedQuoteOptions.map((option) => (
+        {sortedQuoteOptions.map((option, idx) => (
           <RouteOption
-            isSelected={option.id === userSelectedQuoteOptionId}
-            onClickOption={() => setUserSelectedQuoteOptionId(option.id)}
+            isSelected={
+              userSelectedQuoteOptionId === undefined
+                ? idx === 0
+                : option.id === userSelectedQuoteOptionId
+            }
+            onClickOption={() => setUserSelectedQuoteOptionId(option?.id)}
             option={option}
-            key={option.id}
+            key={idx}
```

Comment:
Can we use `uniqueId` as the `key`?

```suggestion
            key={option.uniqueId}
```

Reply:
I think you meant option.id, but we can't use that either because it's only available for LiFi.